diff -urN aa/libsoup-2.1.7/libsoup/soup-gnutls.c libsoup-2.1.7/libsoup/soup-gnutls.c
--- aa/libsoup-2.1.7/libsoup/soup-gnutls.c	2004-02-11 17:19:26.000000000 +0100
+++ libsoup-2.1.7/libsoup/soup-gnutls.c	2004-02-22 16:00:07.795638536 +0100
@@ -20,6 +20,7 @@
 #include <glib.h>
 
 #include <gnutls/gnutls.h>
+#include <gnutls/x509.h>
 
 #include "soup-ssl.h"
 #include "soup-misc.h"
@@ -57,7 +58,6 @@
 	}
 
 	if (status & GNUTLS_CERT_INVALID ||
-	    status & GNUTLS_CERT_NOT_TRUSTED ||
 	    status & GNUTLS_CERT_REVOKED)
 	{
 		g_warning ("The certificate is not trusted.");
