--- libsoup-2.1.2/libsoup/soup-socket.h.wiget	2003-12-12 15:55:43.954279912 +0100
+++ libsoup-2.1.2/libsoup/soup-socket.h	2003-12-12 15:56:28.503462805 +0100
@@ -89,8 +89,8 @@
 
 SoupSocketIOStatus  soup_socket_read       (SoupSocket         *sock,
 					    gpointer            buffer,
-					    guint               len,
-					    guint              *nread);
+					    gsize               len,
+					    gsize              *nread);
 SoupSocketIOStatus  soup_socket_read_until (SoupSocket         *sock,
 					    gpointer            buffer,
 					    guint               len,
@@ -101,8 +101,8 @@
 
 SoupSocketIOStatus  soup_socket_write      (SoupSocket         *sock,
 					    gconstpointer       buffer,
-					    guint               len,
-					    guint              *nwrote);
+					    gsize               len,
+					    gsize              *nwrote);
 
 
 #endif /* SOUP_SOCKET_H */
