--- libsoup-2.2.104/libsoup.pc.in.orig	2007-06-01 22:31:34.000000000 +0200
+++ libsoup-2.2.104/libsoup.pc.in	2008-01-13 09:19:33.406713940 +0100
@@ -8,5 +8,6 @@
 Description: a glib-based HTTP library
 Version: @VERSION@
 Requires: glib-2.0 libxml-2.0 @SSL_REQUIREMENT@
+Requires.private: gthread-2.0 gobject-2.0
 Libs: -L${libdir} -lsoup-${SOUP_API_VERSION}
 Cflags: -I${includedir}/libsoup-${SOUP_API_VERSION}
